src/common/ipfs.ts

```diff
@@ -89,12 +85,15 @@ export async function batchUpload(
   var secret = process.env.PINATA_API_SECRET;
   var jwt = process.env.PINATA_API_JWT;
   if (!key || !secret) {
-    // await axios.get("https://nftlabs-ipfs-server-batch.zeet-nftlabs.zeet.app/grant", {headers }).then((res) => {
 
-    await axios.get("http://localhost:3002/grant", { headers }).then((res) => {
-      key = res.data.key;
-      secret = res.data.secret;
-      jwt = res.data.jwt;
+    await fetch("https://upload.nftlabs.co/grant", {
```

Comment:
We need this to go in the `IpfsStorage` class and implemented in the `getUploadToken` method. We only need to return the `jwt`.

This token will only really be used on the folder uploads because those go straight to IPFS. We can call the `getUploadToken` method internally rather than passing it as a param to the `uploadFolder` method.

Reply:
makes sense